src/components/NcActions/NcActions.vue

```diff
@@ -976,6 +976,14 @@ export default {
 			type: Number,
 			default: 0,
 		},
+
+		/**
+		 * The `inert` attribute for the menu toggle button
+		 */
+		inert: {
+			type: Boolean,
+			default: false,
```

Comment:
Will it ever be something else than `false`?

Reply:
It is `inert` in the current fix for `NcAvatar`